buildbot.mariadb.org/master.cfg

```diff
@@ -1212,6 +1212,28 @@ c['builders'].append(
       properties={'rpm_type': 'fedora35'},
       factory=f_rpm_autobake))
 
+c['builders'].append(
+    util.BuilderConfig(name="amd64-fedora-36",
+      workernames=workers["x64-bbw-docker-fedora-36"],
+      tags=["Fedora", "quick", "gcc"],
+      collapseRequests=True,
+      nextBuild=nextBuild,
+      canStartBuild=canStartBuild,
+      locks=getLocks,
+      properties={'mtr_additional_args': protected_branches_mtr_additional_args},
+      factory=f_quick_build))
```

Comment:
Missing worker definition similar to master-galera/master.cfg on lines around 370. I am working into having a unified worker definition and to get rid of the duplicates, but until then it needs to be specified twice.

Reply:
Added workers. Please review if this is correct.